Basic-Car-Maintenance/Documentation.docc/Documentation.md

```diff
@@ -1,5 +1,4 @@
-# Basic-Car-Maintenance
-
+![Basic Car Maintenance](images/BasicCarMaintenance.png)
```

Comment:
This warning shows on this line because you removed the title

> An article is expected to start with a top-level heading title

If you fix the last warning this one still shows up

> Image in document summary will not be displayed


I'm going to accept this for Hacktoberfest but note I won't be merging this PR because it's not correct right now